documentation-site/pages/blog/introducing-react-view/index.mdx

```diff
@@ -0,0 +1,278 @@
+import metadata from './metadata.json';
+import LiveEditor from './live-editor';
+import Layout from '../../../components/layout';
+import Yard from '../../../components/yard/index';
+import {BlogImage, Meta, Caption} from '../../../components/blog';
+
+import {View, PropTypes} from 'react-view';
+import {Button, SIZE} from 'baseui/button';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+React View is an **interactive playground**, **documentation** and **code generator** for your components. You can use it to document your React component APIs, let users to live-edit the props and grab the generated ready-to-use source code.
+
+The example below demonstrates all basic features of React View:
+
+- The component **live preview**. (`baseui/button` in this case)
+- Knobs representing `Button` props. Fully **interactive**.
+- The **source code editor**. The source code is automatically generated based on the initial configuration and playground state. The source code is **also editable**.
+- Some additional actions. For example, you can **format the code with Prettier** or reset the playground state.
+
+<div style={{margin: '2em 0.5em'}}>
+  <Yard
+    componentName="Button"
+    placeholderHeight={56}
+    theme={[]}
+    props={{
+      children: {
+        value: 'Hello',
+        type: PropTypes.ReactNode,
+        description: 'Visible label.',
+      },
+      size: {
+        value: 'SIZE.default',
+        defaultValue: 'SIZE.default',
+        options: SIZE,
+        type: PropTypes.Enum,
+        description: 'Defines the size of the button.',
+        imports: {
+          'baseui/button': {
+            named: ['SIZE'],
+          },
+        },
+      },
+      onClick: {
+        value: '() => alert("click")',
+        type: PropTypes.Function,
+        description: 'Function called when button is clicked.',
+      },
+      disabled: {
+        value: false,
+        type: PropTypes.Boolean,
+        description: 'Indicates that the button is disabled',
+      },
+      overrides: {
+        value: undefined,
+        type: PropTypes.Custom,
+        description: 'Lets you customize all aspects of the component.',
+        custom: {
+          names: [],
+          sharedProps: {},
+        },
+      },
+    }}
+    scope={{
+      Button,
+      SIZE,
+    }}
+    imports={{
+      'baseui/button': {
+        named: ['Button'],
+      },
+    }}
+  />
+</div>
+
+All these parts work in harmony. An update to one of them synchronizes the state across all of them. So you can change a knob to update the source code or edit the source code to update knobs.
```

Comment:
"All these parts work in harmony—an update to one of them synchronizes the state across all of them"